src/nvim/testdir/test_window_cmd.vim

```diff
@@ -1027,9 +1028,10 @@ func Test_win_move_statusline()
     call assert_equal(h0, winheight(0))
     call assert_equal(1, &cmdheight)
   endfor
-  call assert_true(win_move_statusline(0, 1))
-  call assert_equal(h0, winheight(0))
-  call assert_equal(1, &cmdheight)
+  " Nvim supports cmdheight=0
+  " call assert_true(win_move_statusline(0, 1))
+  " call assert_equal(h0, winheight(0))
+  " call assert_equal(1, &cmdheight)
```

Comment:
can the tests be changed instead of just commented out?

Reply:
OK.